packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -976,6 +993,7 @@ abstract class ResidentRunner {
         printStatus('To toggle the widget inspector (WidgetsApp.showWidgetInspectorOverride), press "i".');
         printStatus('To toggle the display of construction lines (debugPaintSizeEnabled), press "p".');
         printStatus('To simulate different operating systems, (defaultTargetPlatform), press "o".');
+        printStatus('To enable profiling of widget build times, (debugProfileWidgetBuilds), press "a".');
```

Comment:
The dartdoc for `debugProfileWidgetBuilds` is 'Adds [Timeline] events for every Widget built'.

Perhaps, in order to keep this description terse, we say: 'Add timeline events for every widget build method'? Or, 'To enable timeline events for all widget build method, (debugProfileWidgetBuilds), press "a"'?

Reply:
This makes sense to expose now that we have the timeline page in devtools to display the data right next to the button to toggle it on and off.

Reply:
Done